### PR TITLE
Many and various fixes to the feedback tools

### PR DESCRIPTION
**Pull request by pavpanchekha**
This PR fixes a lot of small issues, like comments in a narrow window, not shadowing Ctrl-E on macOS, not showing the pop-up if it's already active, and so on.

### CODE REVIEW

**Comment by pavpanchekha** [comment]
Frankly this code is pretty bad and should be rewritten, but for now this at least fixes some major issues.

File: infra/api.py [hunk 1]
```diff
@@ -249,10 +253,15 @@ def status():
...
-        good = f.read(256)
-    # Equivalent to `good == heng.digest()` but constant-time-ish
-    if sum([0 if a == b else 1 for a, b in zip(good, heng.digest())]) == 0:
+    try:
+        with open("pw.hash", "rb") as f:
+            good = f.read(256)
+        # Equivalent to `good == heng.digest()` but constant-time-ish
+        allowed = sum([0 if a == b else 1 for a, b in zip(good, heng.digest())]) == 0
+    except FileNotFoundError:
+        allowed = True
```
Comment (chrishtr) [inline infra/api.py:262]: What effect does this have? Is it a security hole perhaps?

File: www/feedback.js [hunk 2]
```diff
@@ -1,22 +1,45 @@
...
+    if (navigator.platform.indexOf("Mac") === 0 || navigator.platform === "iPhone") {
+        return e.metaKey;
+    } else {
+        return e.ctrlKey;
+    }  
+}
+
+function ctrl_key_name() {
+    if (navigator.platform.indexOf("Mac") === 0 || navigator.platform === "iPhone") {
+        return "Ctrl";
```
Comment (chrishtr) [inline www/feedback.js:17]: Is this the opposite of what you want?

File: infra/api.py (hunk 1 above)
Comment (pavpanchekha) [reply infra/api.py]: Ok, I've made it so you have to pass a `--no-password` flag.

File: www/feedback.js (hunk 2 above)
Comment (pavpanchekha) [reply www/feedback.js]: Good catch, fixed.

**Pull request merged**